### DISCUSSION TITLE
input helper documentation

### DISCUSSION THREAD

**Discussion by fizzgig314** [pods-framework/pods#6827] [Q&A]
posting this as I'm sure others will have the same question I did.

after migrating to PODS 2 I need to rewrite my input helpers.  any documentation would be greatly appreciated.  thanks.

**Comment by hotzeplotz** [comment]
is this issue really closed? unless i'm missing something obvious, there is no input helper documentation on the new pods.io website for Pods 2.x and i couldn't even find an example of an input helper for Pods 2.x, which could at least get a developer started with porting pre-2.x input helpers.

**Comment by sc0ttkclark** [comment]
We're working on a tutorial for creating new field types and overriding existing field type inputs. Input Helpers are going away, we're suggesting Custom Field Type and Field Input overrides (via WP filter).

**Comment by hotzeplotz** [comment]
thanks @sc0ttkclark - i did some reverse engineering of variable available in v2.x input helpers to re-enable my old input helpers, but indeed it feels much more complicated than it needs to be: custom field types and WP filters would be a great help for advanced admin pages.

since this github issue comes up among top hits for searches for "pods framework input helpers", it may be worth closing this issue with a link to relevant documentation on pods.io when this is available!
